scripts/download_models.py

```
@@ -95,13 +95,89 @@ def create_directory_structure(self):
+            else:
+                print("❌ No verification file found in any location!")
+                print("🔍 Searching for JSON files in likely /workspace subdirectories (this may take a moment)...")
+                try:
+                    # Limit search to specific subdirectories and reduce maxdepth to avoid issues in large filesystems
+                    search_dirs = ['/workspace', '/workspace/models', '/workspace/data', '/workspace/ComfyUI']
+                    per_dir_timeout = int(os.getenv("FIND_TIMEOUT_SECONDS", "10"))
+                    max_list = int(os.getenv("FIND_MAX_RESULTS", "100"))
+                    found_files = []
+                            # Use os.walk to traverse up to depth 2
+                            for root, dirs, files in os.walk(d):
+                                # Calculate depth relative to the search directory
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)